INFO:comtypes.client._code_cache:Could not import comtypes.gen, trying to create it.
INFO:comtypes.client._code_cache:Created comtypes.gen directory: 'C:\Python312\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._code_cache:Writing __init__.py file: 'C:\Python312\Lib\site-packages\comtypes\gen\__init__.py'
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Python312\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._generate:Could not import comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4: No module named 'comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4'
INFO:comtypes.client._generate:# Generating comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4
INFO:comtypes.client._generate:Could not import comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0: No module named 'comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0'
INFO:comtypes.client._generate:# Generating comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0
INFO:comtypes.client._generate:Could not import comtypes.gen.stdole: No module named 'comtypes.gen.stdole'
INFO:comtypes.client._generate:# Generating comtypes.gen.stdole
INFO:comtypes.client._generate:Could not import comtypes.gen.SpeechLib: No module named 'comtypes.gen.SpeechLib'
INFO:comtypes.client._generate:# Generating comtypes.gen.SpeechLib
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Python312\Lib\site-packages\comtypes\gen'
INFO:root:[2023-12-25 14:12:33.833399] Aadhya started.
INFO:comtypes.client._code_cache:Could not import comtypes.gen, trying to create it.
INFO:comtypes.client._code_cache:Created comtypes.gen directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._code_cache:Writing __init__.py file: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen\__init__.py'
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._generate:Could not import comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4: No module named 'comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4'
INFO:comtypes.client._generate:# Generating comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4
INFO:comtypes.client._generate:Could not import comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0: No module named 'comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0'
INFO:comtypes.client._generate:# Generating comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0
INFO:comtypes.client._generate:Could not import comtypes.gen.stdole: No module named 'comtypes.gen.stdole'
INFO:comtypes.client._generate:# Generating comtypes.gen.stdole
INFO:comtypes.client._generate:Could not import comtypes.gen.SpeechLib: No module named 'comtypes.gen.SpeechLib'
INFO:comtypes.client._generate:# Generating comtypes.gen.SpeechLib
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Users\\Aditya\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:root:[2023-12-25 14:37:32.626205] Aadhya started.
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Users\\Aditya\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Users\\Aditya\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Users\\Aditya\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:root:[2023-12-25 14:43:40.003454] Aadhya started.
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Users\\Aditya\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:root:[2023-12-25 14:46:47.670961] Aadhya started.
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Users\\Aditya\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:root:[2023-12-25 14:48:54.677298] Aadhya started.
INFO:comtypes.client._code_cache:Imported existing <module 'comtypes.gen' from 'C:\\Users\\Aditya\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\comtypes\\gen\\__init__.py'>
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'C:\Users\Aditya\AppData\Local\Programs\Python\Python312\Lib\site-packages\comtypes\gen'
INFO:root:[2023-12-25 14:56:15.225379] Aadhya started.
INFO:comtypes.client._code_cache:Could not import comtypes.gen, trying to create it.
INFO:comtypes.client._code_cache:Created comtypes.gen directory: 'D:\Voice-Assistant-in-Python\venv\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._code_cache:Writing __init__.py file: 'D:\Voice-Assistant-in-Python\venv\Lib\site-packages\comtypes\gen\__init__.py'
INFO:comtypes.client._code_cache:Using writeable comtypes cache directory: 'D:\Voice-Assistant-in-Python\venv\Lib\site-packages\comtypes\gen'
INFO:comtypes.client._generate:Could not import comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4: No module named 'comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4'
INFO:comtypes.client._generate:# Generating comtypes.gen._C866CA3A_32F7_11D2_9602_00C04F8EE628_0_5_4
INFO:comtypes.client._generate:Could not import comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0: No module named 'comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0'
INFO:comtypes.client._generate:# Generating comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0
INFO:comtypes.client._generate:Could not import comtypes.gen.stdole: No module named 'comtypes.gen.stdole'
INFO:comtypes.client._generate:# Generating comtypes.gen.stdole
INFO:comtypes.client._generate:Could not import comtypes.gen.SpeechLib: No module named 'comtypes.gen.SpeechLib'
INFO:comtypes.client._generate:# Generating comtypes.gen.SpeechLib
INFO:root:[2023-12-25 14:58:18.597995] Aadhya started.
